### DISCUSSION TITLE
Default column values

### DISCUSSION THREAD

**Discussion by esciara** [TobikoData/sqlmesh#3288] [Q&A] [answered]
Hi,

I could not find a way to define default column values (nothing came out of my searches). Is there a way to set it?

Em

**Comment by georgesittas** [comment, marked as answer]
A bit late to reply here, but perhaps an `ALTER TABLE @this_model ALTER COLUMN ... SET DEFAULT ...` post-statement does the trick. Make sure to run that only during table creation time:

```sql
@IF(
  @runtime_stage = 'creating',
  ALTER TABLE ...
);
```
